src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingStonemason.java

```diff
@@ -50,6 +56,30 @@ public BuildingStonemason(final IColony c, final BlockPos l)
         super(c, l);
     }
 
+    @Override
+    public void checkForWorkerSpecificRecipes()
+    {
+        final IRecipeStorage sandstoneStorage = StandardFactoryController.getInstance().getNewInstance(
+          TypeConstants.RECIPE,
+          StandardFactoryController.getInstance().getNewInstance(TypeConstants.ITOKEN),
+          ImmutableList.of(new ItemStack(Blocks.COBBLESTONE, 1), new ItemStack(Blocks.SAND, 1)),
+          1,
+          new ItemStack(Blocks.SANDSTONE, 1),
+          Blocks.AIR);
```

Comment:
We wanted to make it 2 sandstone here. Could make that a research though too.

Reply:
think the research isnt that required though, since the sand from the crusher already hasone

Reply:
I like the idea of a research for that. This already reduces the cost of sandstone quite a bit, from 4 sand to 1 sand and 1 cobble. Dropping it to 1:1 seems like a research. I'll look at that in a future round.